### ISSUE TITLE
Not building on Mac

### ISSUE THREAD

**Issue by fa-bien** [odow/MathOptFormat.jl#84]
I get the following error when `using MathOptFormat` on MacOS:

```
ERROR: LoadError: LoadError: error compiling doc!: error compiling #handle_message#2: could not load library "libz"
```

As found [here](https://github.com/JuliaIO/GZip.jl/issues/54) it is a bug of GZip.jl; it can be fixed by first having these two lines:
```Julia
using Libdl
push!(Libdl.DL_LOAD_PATH,"/usr/lib")
```
However this is not satisfying on the long term. One suggested solution is to use CodecZlib.jl instead, would that be acceptable?

**Comment by odow** [comment]
Sure. Want to make a pull request?

There are the only lines that need changing:
https://github.com/odow/MathOptFormat.jl/blob/0dcd30db8c1a95555036714b89d0b056ce7a29db/src/MathOptFormat.jl#L136-L142

**Comment by fa-bien** [comment]
PR here https://github.com/odow/MathOptFormat.jl/pull/85

**Comment by odow** [comment]
Closed by #85
